build.xml

```diff
@@ -147,10 +147,10 @@
   </target>
 
   <target name="run" depends="jar"
-          description="Gives Instructions on how to run">
+          description="Gives instructions on how to run">
           <echo message="To run the demonstration:"/>
-          <echo message="Build using 'ant jar'"/>
-          <echo message="Run using 'java -jar ${jar.dir}\${productname}.jar'"/>
+          <echo message="Build using 'ant Run'"/>
```

Comment:
This appears to be circular (running the "run" task now advises running the "Run" task) unless capitalization is significant and a new task named "Run" had been created. (Which isn't the intent.)